src/views/v2/emails/template-parts/body/event/links/gcal.php

```diff
@@ -0,0 +1,34 @@
+<?php
+/**
+ * Event Tickets Emails: Main template > Body > Event > Links > Google Calendar.
+ *
+ * Override this template in your own theme by creating a file at:
+ * [your-theme]/tribe/events/v2/emails/template-parts/body/event/links/gcal.php
+ *
+ * See more documentation about our views templating system.
+ *
+ * @link https://evnt.is/tickets-emails-tpl Help article for Tickets Emails template files.
+ *
+ * @version TBD
+ *
+ * @since TBD
+ *
+ * @var WP_Post $event The event post object with properties added by the `tribe_get_event` function.
```

Comment:
The goal was to follow the pattern we use on any other of the v2 views, where the `$event` is defined, so that people working on customizations know that they have the `$event` available if they want to do anything. 

If you check any other template (views) you'll see this is done. That's why I kept the docblock for the `$event` var, because it's helpful from a devex perspective.

Reply:
>If you check any other template (views) you'll see this is done.
In all 73 of them in TEC, if there is actual code there it's used or explicitly passed. 

Sure, it's useful but it's essentially _global_ - if you are working in event templates `$event` is present.

Anyway, I'm not going to argue about it - it's _available_, so you're listing it. Much like above. I disagree but it's not enough for me to stall this PR.